LegendaryExplorer/LegendaryExplorer/Tools/PackageEditor/Experiments/PackageEditorExperimentsO.cs

```diff
@@ -1061,5 +1961,118 @@ private static StructProperty GenerateExpressionGUID()
 
             return new StructProperty("Guid", props, "ExpressionGUID", true);
         }
+
+        private static void ShowError(string errMsg)
+        {
+            MessageBox.Show(errMsg, "Warning", MessageBoxButton.OK);
+        }
+
+        /// <summary>
+        /// Prompts the user for an int, verifying that the int is valid.
+        /// </summary>
+        /// <param name="msg">Message to display for the prompt.</param>
+        /// <param name="err">Error message to display.</param>
+        /// <param name="biggerThan">Number the input must be bigger than. If not provided -2,147,483,648 will be used.</param>
+        /// <param name="title">Title for the prompt.</param>
+        /// <returns>The input int.</returns>
+        private static int promptForInt(string msg, string err, int biggerThan = -2147483648, string title = "")
+        {
+            if (PromptDialog.Prompt(null, msg, title) is string stringPrompt)
+            {
+                int intPrompt;
+                if (string.IsNullOrEmpty(stringPrompt) || !int.TryParse(stringPrompt, out intPrompt) || !(intPrompt > biggerThan))
+                {
+                    MessageBox.Show(err, "Warning", MessageBoxButton.OK);
+                    return -1;
+                }
+                return intPrompt;
+            }
+            return -1;
+        }
+
+        /// <summary>
+        /// Gets a union of two strings. Assumes both only difer at the end.
+        /// </summary>
+        /// <param name="s1">First string.</param>
+        /// <param name="s2">Second string.</param>
+        /// <returns>Union of input strings.</returns>
+        private static string GetUnionOfStrings(string s1, string s2)
```

Comment:
A more descriptive name for this would be GetCommonPrefix

Reply:
Thanks, that's definitely better